domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -254,6 +254,7 @@ class TopicListController @Inject constructor(
     completionTimeFilter: (Long) -> Boolean
   ): List<PromotedStory> {
 
+    var numberOfDaysPassed: Long
```

Comment:
Can this just be a val in the places it's needed rather than having a higher-scoped var like this?

Reply:
Done.

Reply:
Done.